spec/models/legal_aid_application_spec.rb

```diff
@@ -1745,6 +1745,65 @@
     end
   end
 
+  describe "#associated_applications" do
```

Comment:
Please see comment and suggestion for #lead_application spec.

Reply:
👍 have amended as per suggestion